experimental/segmenter_lstm/Cargo.toml

```diff
@@ -26,9 +26,9 @@ include = [
 [dependencies]
 icu_provider = { version = "0.6", path = "../../provider/core", features = ["macros"] }
 litemap = { version = "0.4.0", path = "../../utils/litemap", features = ["serde"] }
-serde = { version = "1.0", default-features = false, features = ["derive", "std"] }
-serde_json = { version = "1.0", default-features = false, features = ["std"] }
-ndarray = { version = "0.15", features = ["serde"] }
+serde = { version = "1.0", default-features = false, features = ["derive", "alloc"] }
+serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
+ndarray = { git = "https://github.com/rust-ndarray/ndarray", rev = "31244100631382bb8ee30721872a928bfdf07f44", features = ["serde"] }
```

Comment:
hm, would prefer to not have to do this, but I've asked them for a release. fine with doing this temporarily.